### ISSUE TITLE
Edit subfolder: Input fields regarding METS should be hidden, when files are not linked in METS file

### ISSUE THREAD

**Issue by oliver-stoehr** [kitodo/kitodo-production#5693]
When editing a subfolder on the process edit page the two input fields `METS filegroup` and `METS path` are only relevant when the files are linked in the mets file.
When the input field `Link in METS file` is set to `No` the two depended fields should be disabled and/or hidden.

![Bildschirmfoto 2023-06-08 um 16 13 51](https://github.com/kitodo/kitodo-production/assets/32509444/0bc86ad7-d5a3-40d9-a88b-2e3e3587461e)

**Comment by henning-gerhardt** [comment]
Is this a really good idea to deactivate them? How did you select which file group should hold the preview and media view data file information?

I think in this dialog are to things grouped: one thing is what is needed for the application itself and the other thing is which data is needed on export. Both information are not equal in any case and should handled in separate ways.

**Comment by solth** [comment]
> Is this a really good idea to deactivate them? How did you select which file group should hold the preview and media view data file information?

If I am not mistaken, preview and media views are set to **folders**, not to **file groups**. So it should - in theory - make no difference to have subfolders without file groups, should it?

**Comment by henning-gerhardt** [comment]
> > Is this a really good idea to deactivate them? How did you select which file group should hold the preview and media view data file information?
> 
> If I am not mistaken, preview and media views are set to **folders**, not to **file groups**. So it should - in theory - make no difference to have subfolders without file groups, should it?

Yes, you are selecting one file group for media view and a possible other file group for the preview. The information from the file group is used as directory on the one side and the name of the file group for storing it in the internal mets file. But if you can not choose a name of a file group if the usage is set to "no", which name is used in the internal mets file?

On other thing is: with this change the old values for name of the METS filegroup and METS path are still used. Should they not be nulled / cleaned in the database to they can not used any more?

**Comment by solth** [comment]
> Yes, you are selecting one file group for media view and a possible other file group for the preview. The information from the file group is used as directory on the one side and the name of the file group for storing it in the internal mets file. But if you can not choose a name of a file group if the usage is set to "no", which name is used in the internal mets file?

Setting a file group for a subfolder is optional in Kitodo 3 anyway, so this pr would not change anything concerning folders without file groups. If I recall correctly, no file group at all is set for such folders in the internal meta.xml file.

> On other thing is: with this change the old values for name of the METS filegroup and METS path are still used. Should they not be nulled / cleaned in the database to they can not used any more?

Thanks for pointing this out 👍 , I will change this.
